Policies/github-terms-of-service.md

```diff
@@ -36,15 +36,24 @@ Read this Terms of Service agreement carefully before accessing or using GitHub.
 | [Q. Changes to these Terms of Service](#q-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of changes that affect your rights. |
 | [R. Miscellaneous](#r-miscellaneous) | Please see this section for legal details including our choice of law. |
 
+</details>
+
 ## The GitHub Terms of Service
 Effective date: November 13, 2019
 
 ### Section Headings and Summaries
-Throughout this Agreement, each section includes titles and sometimes brief summaries of the following terms and conditions. These section titles and brief summaries (summaries are preceded by the text 'Short version:') are not legally binding—you can completely ignore the summaries to reduce your reading. 
+The different sections in this Agreement have titles and sometimes also brief summaries. Such titles and brief summaries are not legally binding. Each such summary takes the form of a collapsible text that you can expand or collapse by clicking upon the summary's heading, where the summary's heading is always the text 'Click for short version'. Each summary appears just after the relevant section title, and immediately before the relevant section body. You can completely ignore such summaries. 
 
 ### A. Definitions
+<details><summary open="false"><b>Click for short version</b> </summary>
+
 **Short version:** *We use these basic terms throughout the agreement, and they have specific meanings. You should know what we mean when we use each of the terms. There's not going to be a test on it, but it's still useful information.*
 
+</details>
+
+<details>
```

Comment:
:-1

Reply:
No problem with not implementing this suggested collapsible-text implementation.

Thought it might be useful because after reading definitions sections, I often just skip them except when I want clarity on a particular term.

If it's possible, can you explain why you think this is a bad idea?

Reply:
I mean it makes no sense to collapse whole sections. ToS is applied as whole. Instead I suggest to summarize briefly a section and to put under a collapsible block additional details. And make all the blocks expanded by default - if a user doesn't need them, he can collapse them with a single click.

Reply:
Okay.

I don't mind about removing the collapsible block that covers the whole of the Definitions section.

I disagree with making blocks expanded by default. I don't see the point. I want to incline users to less reading, and part of that is making the blocks closed by default. I've read many user agreements, and in the vast majority of cases, for general consumers (such as might be the majority of GitHub users), the limitation on liability as well as some other sections are hardly at all important. Based on the wisdom of knowing this, and acting in the role of indirect adviser to GitHub users, I want to incline users toward not reading the detail of such things. Also, having the blocks open by default makes the whole agreement look more detailed, complicated and lengthy on first perusal. I deliberately refuse using services on occasion based on services having overly long contracts. I want to encourage potential GitHub users as much as possible into not being disheartened by the appearance of overly long and complicated terms. Blocks being closed by default is part of this.

Just a few notes about the summaries. Unfortunately, the present summaries to my mind, are a complete waste of time. The main problem with them is that they are not always accurate. Sometimes important details are missing. If you can't know whether a particular summary is accurate, you are forced into reading the whole section. It would be better to remove all the summaries, or to indicate that all the summaries are accurate along with making them accurate.